src/NHibernate/Hql/Ast/ANTLR/Tree/ASTNode.cs

```diff
@@ -433,35 +433,33 @@ object ITree.DeleteChild(int i)
 
 		void ITree.ReplaceChildren(int startChildIndex, int stopChildIndex, object t)
 		{
-            if (_children != null)
-            {
-                _children.RemoveRange(startChildIndex, stopChildIndex - startChildIndex + 1);
-            }
-            if (_children == null)
-            {
-                _children = new List<IASTNode>(); 
-            }
-
-            IASTNode node = t as IASTNode;
-
-            if (node != null)
-            {
-                _children.Insert(startChildIndex, (IASTNode)t);
-            }
-            else
-            {
-                IEnumerable list = t as IEnumerable;
-
-                if (list != null)
-                {
-                    int i = 0;
-                    foreach (IASTNode entry in list)
-                    {
-                        _children.Insert(startChildIndex + i, entry);
-                        i++;
-                    }
-                }
-            }
+			if (_children != null)
+			{
+				_children.RemoveRange(startChildIndex, stopChildIndex - startChildIndex + 1);
+			}
+			if (_children == null)
+			{
+				_children = new List<IASTNode>();
+			}
+
+			if (node != null)
```

Comment:
@pawan-deepsource, here the variable is missing.

Reply:
I'm taking a look at this too.

Reply:
Here's the generated code after applying the patch to the Autofix: https://github.com/pawan-deepsource/nhibernate-core/pull/1/commits/e6bd5b611eec547efba42522a9b547ea3257816e?